packages/core/package.json

```diff
@@ -39,17 +38,12 @@
     "@truffle/migrate": "^3.2.31",
     "@truffle/plugins": "^0.2.3",
     "@truffle/preserve": "^0.2.3",
-    "@truffle/preserve-fs": "^0.2.3",
-    "@truffle/preserve-to-buckets": "^0.2.3",
-    "@truffle/preserve-to-filecoin": "^0.2.3",
-    "@truffle/preserve-to-ipfs": "^0.2.3",
     "@truffle/provider": "^0.2.33",
     "@truffle/provisioner": "^0.2.21",
     "@truffle/require": "^2.0.66",
     "@truffle/resolver": "^7.0.16",
     "@truffle/source-fetcher": "^0.5.1",
     "@truffle/workflow-compile": "^3.2.17",
-    "app-module-path": "^2.2.0",
```

Comment:
This needs to be made a devDep because it's [used](https://github.com/trufflesuite/truffle/blob/develop/packages/core/test/lib/run.js#L15) by tests

Reply:
moved to devDeps